File: backend spring/src/main/java/com/bloodbank/dto/response/DashboardStatsDto.java
package com.bloodbank.dto.response;

import lombok.Builder;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
@Builder
public class DashboardStatsDto {
    private long totalDonors;
    private long totalHospitals;
    private long totalPendingRequests;
    private long totalBloodUnits;
}

File: BackendDotNet/DTOs/Response/DashboardStatsDto.cs
namespace BackendDotNet.DTOs.Response
{
    public class DashboardStatsDto
    {
        public int TotalDonors { get; set; }
        public int TotalHospitals { get; set; }
        public int PendingRequests { get; set; }
        public int TotalBloodUnits { get; set; }
    }
}
